src/popup/router/components/InviteItem.vue

```diff
@@ -1,36 +1,58 @@
 <template>
   <div class="invite-row">
     <div class="invite-info">
-      <span class="primary">{{ balance }} {{ $t('pages.appVUE.aeid') }}</span>
+      <span class="primary">
+        <span class="balance">{{ balance }}</span>
+      </span>
       <!--eslint-disable-line vue-i18n/no-raw-text-->
-      ({{ (balance * current.currencyRate).toFixed(2) }}
+      ({{ currencySigns[current.currency.toUpperCase()] }}
       <!--eslint-disable-next-line vue-i18n/no-raw-text-->
-      {{ current.currency.toUpperCase() }})
+      {{ (balance * current.currencyRate).toFixed(2) }})
```

Comment:
There is a similar part in the TransactionItem component, shouldn't we extract it?
https://github.com/aeternity/superhero-wallet/blob/5b779c30840c2e4fdaf9086e9691e7b30149883d/src/popup/router/components/TransactionItem.vue#L4-L13